services/forms-web-app/src/server.js

```diff
@@ -3,10 +3,9 @@
 /**
  * Module dependencies.
  */
-
-const debug = require('debug')('appeal-planning-decision-ux:server');
 const http = require('http');
-const app = require('../app.js');
+const logger = require('pino')();
```

Comment:
We'll need to configure this properly, but can do that afterwards

Reply:
👍